docs/getting_started.rst

```diff
@@ -181,49 +185,41 @@ advanced commands to interact with memote repositories.
 Offline, local or private workflow:
 -----------------------------------
 
-Users that have decided to not to use `GitHub`_ (or `GitLab`_ **Not implemented yet**), or those that
-have decided to set up a private model repository, will need to manually 
-execute the commands that would otherwise be carried out by the continuous 
-integration server after each modification of the model:
+Users that have decided to not to use `GitHub`_ (or `GitLab`_ **Not implemented
+yet**), or those that have decided to set up a private model repository, will
+need to manually execute the commands that would otherwise be carried out by the
+continuous integration server after each modification of the model:
 
 In sequence, these are:
 
-1. .. code-block:: console
-
-      $ git add .
+1. On the desired branch make a change to the model and commit that change.
 
-2. .. code-block:: console
+   .. code-block:: console
 
+      $ git add .
       $ git commit -m <Some Description>
 
-3. .. code-block:: console
+2. After committing your changes run memote
```

Comment:
Point two now reads: 2. After committing your changes run memote `memote run`.

With the first "memote" outside a code block and the second inside. Did you mean to keep the first "memote"?

Reply:
Reworded to be avoid duplication.